src/test/java/horseshoe/RunnerTests.java

```diff
@@ -151,7 +151,7 @@ public void testHelp() throws IOException {
 
 	@Test (expected = Test.None.class) // No exception expected
 	public void testJson() throws IOException {
-		assertEquals(0L, Files.walk(Paths.get("samples/json"))
+		assertEquals(0L, Files.walk(Paths.get("samples/data/json"))
```

Comment:
There should be a couple other places that need updated paths.

Reply:
Should be fixed now.